lib/ansible/modules/crypto/openssl_privatekey.py

```diff
@@ -123,6 +123,31 @@
         default: auto
         choices: [ auto, cryptography, pyopenssl ]
         version_added: "2.8"
+    format:
+        description:
+            - Determines which format the private key is written in. By default, PKCS1 (traditional OpenSSL format)
+              is used for all keys which support it. Please note that not every key can be exported in any format.
+            - The value C(auto) selects a backend depending on the key format. The value C(auto_ignore) does the same,
+              but for existing private key files, it will not force a regenerate when its format is not the automatically
+              selected one for generation.
+            - Note that if the format for an existing private key mismatches, the key is *regenerated* by default.
+              To change this behavior, use the I(format_mismatch) option.
+            - Only supported by the C(cryptography) backend.
```

Comment:
I would be more specific on what is only supported by the cryptography backend

Reply:
Good idea. I've extended the text, and also added what the `pyopenssl` backend does (fail if value is not `auto_ignore`).